public/scss/inc/mixins.scss

```diff
@@ -57,6 +56,6 @@
   (-webkit-min-device-pixel-ratio: 1.3),
   (min-device-pixel-ratio: 1.3),
   (min-resolution: 1.3dppx) {
-    background-image: image-url("#{$filename}@2x.#{$extension}");
```

Comment:
An example of something it caught. I'd missed tweaking this line (the mixin image-url was removed in sass3.1)